src/transformers/utils/hub.py

```diff
@@ -221,7 +221,8 @@ def extract_commit_hash(resolved_file: Optional[str], commit_hash: Optional[str]
     """
     if resolved_file is None or commit_hash is not None:
         return commit_hash
-
+    if isinstance(resolved_file, str) and sys.platform == "win32":
+        resolved_file = str(Path(resolved_file).resolve().as_posix())
```

Comment:
```suggestion
    resolved_file = str(Path(resolved_file).resolve().as_posix())
```
No need for the test now :-)

Reply:
This moves the band-aid from the cut to the whole arm, so was just erring on the side of caution.
If that's cool though then all good, added the commit and updated. Thanks!